RescueRS/Presenter/Controllers/App/V1/Rescue/DTOs/RescueDTO.cs

```diff
@@ -36,6 +37,7 @@ public class RescueDTO
             UpdateDateTime = entity.UpdateDateTime,
             Cellphone = entity.ContactPhone,
             Distance = latitude != null && longitude != null ? GetDistanceInMeters(latitude.Value, longitude.Value, entity.Latitude, entity.Longitude) : null,
+            StartedByMe = (entity.Status == RescueStatusEnum.InProgress && entity.UpdatedBy == currentUserId) || (entity.Status == RescueStatusEnum.Completed && entity.ConfirmedBy == currentUserId)
```

Comment:
Since a rescue can be completed by the user who requested it, will the name "StartedByMe" still a clear choice?